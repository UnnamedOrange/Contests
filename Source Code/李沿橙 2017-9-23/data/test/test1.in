5
15
336 452 364 439 432 338 305 367 417 379 345 425 291 416 441 
49
496 485 588 581 557 554 409 534 432 576 425 422 511 425 542 412 463 382 537 505 403 586 439 538 574 583 509 373 481 380 430 573 552 388 557 537 437 564 567 442 551 513 400 473 491 516 390 589 496 
10
313 588 434 374 431 458 373 340 368 526 
8
378 508 377 215 400 492 418 454 
50
300 365 868 513 201 334 897 833 901 482 667 714 97 547 671 712 305 130 283 648 136 262 799 750 532 658 262 304 293 652 177 439 852 602 773 544 874 879 805 332 447 656 490 862 471 945 319 121 337 922 
