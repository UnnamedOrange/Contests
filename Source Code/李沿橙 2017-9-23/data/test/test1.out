291 305 336 345 364 367 416 417 432 439 441 452 425 379 338 
373 380 382 388 390 400 409 412 422 425 425 430 437 439 442 473 481 485 496 496 505 509 511 513 534 537 537 538 551 552 554 557 557 564 567 574 576 581 583 586 588 589 573 542 516 491 463 432 403 
313 340 368 373 374 431 458 588 526 434 
215 377 400 418 454 492 508 378 
97 121 130 136 201 262 283 293 300 304 305 319 332 334 365 447 471 482 490 513 544 547 648 652 656 658 667 712 714 750 799 805 833 852 862 868 879 897 901 922 945 874 773 671 602 532 439 337 262 177 
